.github/pull_request_template.md

```diff
@@ -0,0 +1,63 @@
+<!--
+Thank you for contributing to the MariaDB Server repository!
+
+You can help us review your changes by answering these questions <3
+
+The more information you provide about the contribution, the better
+for everybody. You will be happier because your contribution will be
+reviewed faster :-), the reviewer is happier because more Pull Requests
+can be accepted, other contributors are happier because they wait
+less for their PRs, the community is happier because their project
+gets better, faster.
+
+If you have any questions related to MariaDB or you just want to
+hang out and meet other community members, please join us on
+https://mariadb.zulipchat.com/ .
+-->
+
+<!--
+Please remember to search if there is an existing Jira MDEV task
+for this contribution and if there isn't one, we would highly
+appreciate if you could create one at https://jira.mariadb.org/
+and have its title be identical to the one of the Pull Request.
+Example:
+"MDEV-21221 Fix crash in sql_acl.cc when new user is created"
+-->
+- [ ] *The jira.mariadb.org issue number for this PR is: MDEV-_____*
+
+<!--
+Please tick the appropriate box by adding an "x" between the square brackets.
+-->
+## Type of contribution
+- [ ] Bug fix
+- [ ] New feature (i.e. new functionality that is not expected to affect any other part of the server)
+- [ ] Code refactoring
+- [ ] Documentation
+- [ ] Other
+
+<!--
+An amazing description should answer some questions like:
+1. What problem is the patch trying to solve?
+2. If some output changed, what was it looking like before
+   the change and how it's looking with this patch applied
+3. Do you think this patch might introduce side-effects in
+   other parts of the server?
+-->
+## Description
+TODO: fill description here
+
+<!--
+You might consider answering some questions like:
+1. Does this affect the on-disk format used by MariaDB?
+2. Does this change any behavior experienced by a user who upgrades from a version prior to this patch?
+-->
+## Backward compatibility
+TODO: fill details here if applicable or remove the section
+
+<!--
```

Comment:
Where does the comment show up for a contributor doing a PR?

Reply:
If you look at the screenshots I posted initially, there is an example of how the thing looks when you create a new PR. Basically, the contributor sees a pre-filled PR Description box with the exact content as the template, so the contributor sees the raw markdown text (including comments). Only after the PR is created, the markdown from the template will be rendered (if the contributor chooses not to erase everything, which will definitely happen in some cases 😆  )